Library/Homebrew/language/node.rb

```diff
@@ -4,6 +4,17 @@ def self.npm_cache_config
       "cache=#{HOMEBREW_CACHE}/npm_cache\n"
     end
 
+    def self.pack_for_installation
+      # npm 5.0.0 broke the assumption Homebrew made that npm would always
+      # create and install real directories when passed a file path. We can
+      # restore the old behaviour by packing the path first with "npm pack"
+      # which also helpfully spits out the filename it just created allowing
+      # us to pass that as the final argument.
```

Comment:
Ok, something like `npm pack is needed to make a "clean" installation to a new directory i.e. not just install symlinks back to the source directory in the buildpath` will do